docs/pages/admin-guides/management/guides/aws-iam-identity-center.mdx

```diff
@@ -182,11 +173,60 @@ Clicking the "Log In" button for this app takes you to your Identity Center
 SSO start page which you can use to pick a role and connect to your AWS account
 as usual.
 
+## Background: Account Assignments
+
+The main resource that the Teleport Identity Center integration manages is the
+*Account Assignment*.
+
+An Account Assignment is the combination of a specific Permission Set on a specific
+AWS account - for example "*AdminAccess on Production*" (where *Production* is 
+an AWS account managed by Identity Center). 
+
+When a user has access to an Account Assignment in Teleport, that access is
+mirrored in AWS Identity Center. When a teleport user loses access to an Account
+Assignment in Teleport, that access is similarly deleted in AWS. 
+
+Access to Account Assignments can be granted via Teleport Roles, either directly
+to Users or through Access Lists, or by Account Assignment resources included in
+an approved Access Request.
```

Comment:
It is a bit hard to grasp the Account Assignments concept without introducing concept of yaml account assignment object that is mentioned later: https://github.com/gravitational/teleport/pull/50977/files#diff-fb2062905f79b4f115956784c5dff3435d4e742630bdca6acd0d2a68abc1e996L249